# HG changeset patch
# User Rik <rik@octave.org>
# Date 1506464288 25200
#      Tue Sep 26 15:18:08 2017 -0700
# Branch stable
# Node ID a6cf8e06603a2632f9a9f23435a3c497896fe547
# Parent  ae35d50d975673e6a22f79cea912c7edfdd0b1eb
doc: Correct documentation for randg (bug #52118).

* rand.cc (Frandg): Correct documentation to state that first argument is A, an
input to gamma(), rather than matrix dimensions of return argument.

diff --git a/libinterp/corefcn/rand.cc b/libinterp/corefcn/rand.cc
--- a/libinterp/corefcn/rand.cc
+++ b/libinterp/corefcn/rand.cc
@@ -636,27 +636,28 @@ J. Statistical Software, vol 5, 2000,
 %!   assert (var (x), 1, 0.03);
 %!   assert (skewness (x), 2, 0.06);
 %!   assert (kurtosis (x), 6, 0.7);
 %! endif
 */
 
 DEFUN (randg, args, ,
        doc: /* -*- texinfo -*-
-@deftypefn  {} {} randg (@var{n})
-@deftypefnx {} {} randg (@var{m}, @var{n}, @dots{})
-@deftypefnx {} {} randg ([@var{m} @var{n} @dots{}])
+@deftypefn  {} {} randg (@var{a}, @var{n})
+@deftypefnx {} {} randg (@var{a}, @var{m}, @var{n}, @dots{})
+@deftypefnx {} {} randg (@var{a}, [@var{m} @var{n} @dots{}])
 @deftypefnx {} {@var{v} =} randg ("state")
 @deftypefnx {} {} randg ("state", @var{v})
 @deftypefnx {} {} randg ("state", "reset")
 @deftypefnx {} {@var{v} =} randg ("seed")
 @deftypefnx {} {} randg ("seed", @var{v})
 @deftypefnx {} {} randg ("seed", "reset")
 @deftypefnx {} {} randg (@dots{}, "single")
 @deftypefnx {} {} randg (@dots{}, "double")
+
 Return a matrix with @code{gamma (@var{a},1)} distributed random elements.
 
 The arguments are handled the same as the arguments for @code{rand}, except
 for the argument @var{a}.
 
 This can be used to generate many distributions:
 
 @table @asis
